ert3/engine/_run.py

```diff
@@ -165,6 +166,10 @@ def run(
         .set_type("function" if isinstance(stage, ert3.config.Function) else "unix")
     )
 
+    if local_test_run:
+        local_runpath = Path(workspace._path) / "local-test-run"
```

Comment:
This is misusing the workspace. A way forward could be to extend the workspace such that I could ask for an export folder and then use it 🤷🏻